src/parser/ParsedQuery.h

```diff
@@ -337,21 +337,39 @@ class ParsedQuery {
 
     // Add a variable, that was found in the query body. The added variables
     // will only be used if `isAsterisk` is true.
-    void registerVariableVisibleInQueryBody(const string& variable) {
+    void aux_registerVariableVisibleInQueryBody(const string& variable) {
```

Comment:
What is the `aux_` supposed to mean, was there anything wrong with the previous name?

Reply:
It has the same name as the function which calls it (outside struct SelectedVarsOrAsterisk)
'Aux' is intended to mean 'auxiliary' (function)